---
string json_meta_data
uint8[] rgbd_data
